### ISSUE TITLE
get_cik_map() causes AttributeError due object type "None" in returned list

### ISSUE THREAD

**Issue by niklascube** [sec-edgar/sec-edgar#261]
**Description**
Using the method get_cik_map() of the module cik_lookup an AttributeError with the message "'NoneType' object has no attribute 'upper'" is generated. This error occurs because in the returned list an object has the type NULL. 

Object of matter: `{"cik_str":1104206,"ticker":"RIGG","title":null}`

**To Reproduce**
Just start the method get_cik_map() from module cik_lookup.

**Error**
<img width="1052" alt="image" src="https://user-images.githubusercontent.com/58143162/148091742-4ba1399b-656e-4929-ac26-8562093d6420.png">

**My system**
MacBook Pro 2021

**Comment by niklascube** [comment]
Just saw, somebody already tried to fix the issue by just skipping the entry. I committed something which replaces the None with as an empty string.

**Comment by jackmoody11** [comment]
Hi @niklascube, I think this should be fixed. Can you try using the latest code in master?

**Comment by niklascube** [comment]
> Hi @niklascube, I think this should be fixed. Can you try using the latest code in master?
> 
> See #256.

Not really. The current solution skips the whole object, so that the mentioned RIGG entry is deleted. My solution does not skip the object and returns the object with an empty title. The cik is returned probably then.

**Comment by niklascube** [comment]
I've made a suggestion on my forked repo: https://github.com/niklascube/sec-edgar/commit/495c688cded04324167c4563289e85ff4ead6bce

@jackmoody11 may you have a look?

**Comment by jackmoody11** [comment]
@niklascube if multiple tickers or company names are missing, won't the empty string key be overwritten?
